configs/Wisun_Stm_s2lp_RF.json

```diff
@@ -43,8 +43,26 @@
         "operating-mode": 255,
         "uc-fixed-channel": "0xffff",
         "bc-fixed-channel": "0xffff",
-        "network-name": "\"Wi-SUN Network\""
+        "network-name": "\"Wi-SUN Network\"",
+        "certificate-header": {
+            "help": "Certificate header",
+            "value": "\"wisun_certificates.h\""
+        },
+        "root-certificate": {
```

Comment:
Is it necessary to define certificate names? Could we just use table names, like "WISUN_ROOT_CERTIFICATE" directly in the code?

Reply:
Yes, that is possible also. If .json is used the configuration is more flexible; if there would be several server or root certificates inside the wisun_certificates.h, then by modifying .json file, each server could be configured to have own certificates. Also the configuration system is more visible to users when in .json.